test/tests/performance/pbs_client_nagle_performance.py

```diff
@@ -130,6 +140,37 @@ def test_qdel_nagle_perf(self):
         self.logger.info(
```

Comment:
From the looks of this test, the amount of data between triggering nagle's algorithm and not triggering it is very small.  If we ever add a new server attribute that prints by default, we might push ourselves over that limit again.  I know this isn't your test or even your area of expertise, but I worry about the test's future.  That said, I don't know what to do.  Maybe we should ask for Subhasis's help.